src/test/java/io/stargate/sgv2/jsonapi/service/resolver/CommandResolverWithVectorizerTest.java

```diff
@@ -420,183 +413,6 @@ public void findOneAndDelete() throws Exception {
               });
     }
 
-    @Test
-    public void findOneAndReplace() throws Exception {
```

Comment:
Instead of removing can we change the test to show how the object look like when `$vectorize` is used?

Reply:
Same as above.